src/helpers/helpers.canvas.js

```diff
@@ -49,103 +49,114 @@ var exports = module.exports = {
 	},
 
 	drawPoint: function(ctx, style, radius, x, y, rotation) {
-		var type, edgeLength, xOffset, yOffset, height, size;
-		rotation = rotation || 0;
+		// call draw Symbol with converted radius to width and height
+		// and move x, y to the top left corner
+		this.drawSymbol(ctx, style, radius * 2, radius * 2, x - radius, y - radius, rotation, true);
+	},
+
+	drawSymbol: function(ctx, style, width, height, x, y, rotation, isPoint) {
 
 		if (style && typeof style === 'object') {
-			type = style.toString();
+			var type = style.toString();
 			if (type === '[object HTMLImageElement]' || type === '[object HTMLCanvasElement]') {
-				ctx.drawImage(style, x - style.width / 2, y - style.height / 2, style.width, style.height);
-				return;
+				ctx.drawImage(style, x, y, style.width, style.height);
+				return false;
 			}
 		}
-
-		if (isNaN(radius) || radius <= 0) {
-			return;
+		if (isNaN(width) || width <= 0) {
+			return false;
+		}
+		var vx, vy;
+		if (rotation) {
+			ctx.save();
+			ctx.translate(x + width / 2, y + width / 2);
+			ctx.rotate(rotation * Math.PI / 180);
+			ctx.translate(-width / 2, -width / 2);
+			vx = 0;
+			vy = 0;
+		} else {
+			vx = x;
+			vy = y;
 		}
+		var radius = width / 2;
+		var yRadius = height / 2;
+		var padLeft = isPoint ? radius - width / 2 / Math.sqrt(2) : 0;
```

Comment:
can you add a comment explaining why padding is necessary and what `isPoint` is doing? I'm not sure what that part is for

Reply:
I added comments on that. In the production version some symbols can only be called as point and they are not usign the full width since the size is based on the radius. When is used as a legend, I want to be sure that the symbol width = the width value pass as parameter. So to not impact the current behavior (when symbol is called for a point) I did that logic for the impacted symbols. I.e. Add padding when called as a point (not usign the full width) and use the full width (no padding) when called as a legend symbol.